en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md

```diff
@@ -0,0 +1,318 @@
+               <property expression="json-eval($.resourceType)" name="type" scope="default" type="STRING"/>
+               <property expression="json-eval($.format)" name="format" scope="default" type="STRING"/>
+               <property expression="json-eval($.id)" name="id" scope="default" type="STRING"/>
+               <property expression="json-eval($.summary)" name="summary" scope="default" type="STRING"/>
+               <fhir.init>
+                   <base>{$ctx:base}</base>
+               </fhir.init>
+               <fhir.readSpecificResourceById>
+                   <type>{$ctx:type}</type>
+                   <id>{$ctx:id}</id>
+                   <format>{$ctx:format}</format>
+                   <summary>{$ctx:summary}</summary>
+               </fhir.readSpecificResourceById>
+               <log level="full" separator=","/>
+               <send/>
+           </inSequence>
+           <outSequence/>
+           <faultSequence/>
+       </resource>
+       <resource methods="POST" url-mapping="/delete">
+           <inSequence>
+               <property expression="json-eval($.base)" name="base" scope="default" type="STRING"/>
+               <property expression="json-eval($.resourceType)" name="type" scope="default" type="STRING"/>
+               <property expression="json-eval($.format)" name="format" scope="default" type="STRING"/>
+               <property expression="json-eval($.idToDelete)" name="idToDelete" scope="default" type="STRING"/>
+               <fhir.init>
+                   <base>{$ctx:base}</base>
+               </fhir.init>
+               <fhir.delete>
+                   <type>{$ctx:type}</type>
+                   <idToDelete>{$ctx:idToDelete}</idToDelete>
+               </fhir.delete>
+               <log level="full" separator=","/>
+               <send/>
+           </inSequence>
+           <outSequence/>
+           <faultSequence/>
+       </resource>
+       <resource methods="POST" url-mapping="/update">
+           <inSequence>
+               <property expression="json-eval($.base)" name="base" scope="default" type="STRING"/>
+               <property expression="json-eval($.resourceType)" name="type" scope="default" type="STRING"/>
+               <property expression="json-eval($.format)" name="format" scope="default" type="STRING"/>
+               <property expression="json-eval($.idToUpdate)" name="idToDelete" scope="default" type="STRING"/>
+               <fhir.init>
+                   <base>{$ctx:base}</base>
+               </fhir.init>
+               <fhir.update>
+                   <type>{$ctx:type}</type>
+                   <idToUpdate>{$ctx:idToUpdate}</idToUpdate>
+                   <format>{$ctx:format}</format>
+               </fhir.update>
+               <log level="full" separator=","/>
+               <send/>
+           </inSequence>
+           <outSequence/>
+           <faultSequence/>
+       </resource>
+   </api>
+
+    ```
```

Comment:
Here, point to the reference doc 
To learn about supported operations and their parameters, please refer to `FHIR connector reference`.

Reply:
Added.